modules/hibernate-4.2/README.txt

```diff
@@ -32,17 +32,3 @@ interested in):
     ...
 </project>
 
-
-LGPL dependencies
```

Comment:
and this one

Reply:
On this and other LGPL-related comments.
The text being removed was never true. GridGain of all editions can include LGPL, and always included it in regular editions. Also, we never had any source releases. This text seems to be just a copy-paste from Apache Ignite which was never altered in the fork.

I suggest we proceed with removing LGPL clauses.